pallets/bridge/src/lib.rs

```diff
@@ -84,6 +85,9 @@ pub mod pallet {
         /// Root can always do this.
         type OperateOrigin: EnsureOrigin<Self::Origin>;
```

Comment:
OperateOrigin looks not clear

Reply:
I split it into `UpdateChainOrigin` and `UpdateTokenOrigin`